runtime/doc/options.txt

```diff
@@ -517,10 +517,11 @@ Examples:
 
 The second form (this is compatible with some versions of Vi):
 
-	[text]{white}{vi:|vim:|Vim:|ex:}[white]se[t] {options}:[text]
+	[text{white}]{vi:|vim:|Vim:|ex:}[white]se[t] {options}:[text]
 
-[text]			any text or empty
-{white}			at least one blank character (<Space> or <Tab>)
+[text{white}]		empty or any text followed by at least one blank
+			character (<Space> or <Tab>); "ex:" always requires at
+			least one blank character
```

Comment:
I think that the discussion in
https://groups.google.com/d/topic/vim_dev/-hHiBLao06Y/discussion
should be also applied to the second form.